core/config.go

```diff
@@ -6,6 +6,7 @@ import (
 	"os"
 
 	"github.com/Masterminds/semver/v3"
+	"github.com/creasty/defaults"
```

Comment:
Let's not use a package here, just using the defaults of 0 will be fine. The `init` command can create a good markdown config when creating a new project but I want to adjust how defaults are configured and loaded in the future anyway.